### DISCUSSION TITLE
Need regexp rule to extract title from the url of talk Tiddlywiki post

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#130] [Q&A] [answered]
I need regexp rule to extract only the title of  post from the url of a talk Tiddlywiki post

For example, from the url 
 https://talk.tiddlywiki.org/t/how-big-can-a-tiddlywiki-get/9610

I need to extract only the _how-big-can-a-tiddlywiki-get part_

**Comment by buggyj** [comment, marked as answer]
try this:

```
#match=/^https:\/\/talk\.tiddlywiki\.org\/t\/([^\/]+)[\s\S]*$/
#sub=$1
$title=((*@pageRef/#match/#sub*))
```
